projects/MathJax.md

```diff
@@ -14,7 +14,7 @@ Facebook:  <https://www.facebook.com/mathjax/>
 
 ### Citing
 
-**The recommended way to cite PROJECT in a paper or a presentation is as follows:**
+**The recommended way to cite MathJax in a paper or a presentation is as follows:**
 
 [MathJax](https://www.mathjax.org)
```

Comment:
@dvpc Is there a paper associated with MathJax that people could cite?

Reply:
Well, there is a [short article](http://dx.doi.org/10.1090/noti794) in the *Notices of the American Mathematical Society* that I wrote in 2010, which is the earliest publication about MathJax.  That is probably the best choice, though there have been several technical papers about the assistive technology since then.  Also, the AMS is planning a new *Notices* article giving about the history of the MathJax Project, the AMS’s support, and it becoming part of NumFOCUS, but that hasn't appeared yet (not even written yet, as far as I know)(.

Reply:
Okay, well I might suggest including reference to that article in your "citation" section.